lab9/phoneBook.js

```diff
@@ -0,0 +1,42 @@
+var phoneBook; // Тут ви зберігаєте записи як хочете
+phoneBook = {cards: [], found: []};
+/*
+   Функція додавання запису в телефонну книгу.
+*/
+phoneBook.add = function (name, phone, email) {
+  // Ваша неймовірна магія тут
+  this.cards.push({name, phone, email});
```

Comment:
https://github.com/airbnb/javascript#whitespace--in-braces
{ name, phone, email }